tests/utils.py

```diff
@@ -19,6 +28,39 @@ def get_or_create(self, obj: GraphObject) -> Tuple[GraphObject, bool]:
         return None, True
 
 
+def init_integration_env():
```

Comment:
Maybe it is better to have a `tests_init.py ` file (or something similar), containing the test's init logic. 
I think these functions don't belong in the same file.